CreateMessage/handler.ts

```diff
@@ -286,6 +307,14 @@ export function CreateMessageHandler(
     const { service, email: serviceUserEmail } = userAttributes;
     const { authorizedRecipients, serviceId } = service;
 
+    const canSendAttachments =
+      messagePayload.feature_level_type !== FeatureLevelTypeEnum.ADVANCED &&
```

Comment:
here we are checking that `feature_level_type` is not advanced and the flag `has_attachments` is true, in this case we want to return 403, otherwise the execution can continue